go.mod

```diff
@@ -22,14 +22,14 @@ require (
 	github.com/nhatthm/aferocopy v1.0.2
 	github.com/otiai10/copy v1.6.0
 	github.com/pmezard/go-difflib v1.0.0
+	github.com/qri-io/jsonschema v0.2.1
```

Comment:
Musel som vymenit kniznicu pre JSON schema `xeipuuv/gojsonschema` -> `qri-io/jsonschema`.

Obe kniznice vedia validovat JSON schema.
No iba  v tej novej sa da JSON schema citat a iterovat nad nim.
... a to potrebujem aby som vedel z Json schema vycitat default hodnoty a vytvorit prazdnu konfiguraciu, ktora bude mat hodnoty uz predvyplnene.